### PR TITLE
EVG-14432 Updated tasklog links for execution tasks without tests

### PR DESCRIPTION
**Pull request by bynn**
[EVG-14432](https://jira.mongodb.org/browse/EVG-14432)

### Description 
fixed task logs pointing to wrong link bug

### Testing 
created this ticket on staging 
https://jira.mongodb.org/browse/EVG-14836 (updated to the latest commit)

### CODE REVIEW

File: trigger/task_jira.go [hunk 1]
```diff
@@ -71,42 +69,63 @@ func getTaskURL(data *jiraTemplateData) (string, error) {
...
+				info.Tests = strings.Join(testIDs, " ")
+
+				result = append(result, info)
+			}
+			return result, nil
+		} else {
+			// Task is display only without tests
+			result := make([]taskInfo, 0)
+			for _, taskName := range data.Task.ExecutionTasks {
+				execTask, err := task.FindOneId(taskName)
```
Comment (ablack12) [inline trigger/task_jira.go:105]: If we need to iterate through all exec tasks, can we use FindTasksByIds so we can make just one DB call?

File: trigger/task_jira.go [hunk 2]
```diff
@@ -71,42 +69,63 @@ func getTaskURL(data *jiraTemplateData) (string, error) {
...
+				if err != nil {
+					return nil, err
```
Comment (ablack12) [inline trigger/task_jira.go:107]: can you wrap this error?

File: trigger/task_jira.go [hunk 3]
```diff
@@ -71,42 +69,63 @@ func getTaskURL(data *jiraTemplateData) (string, error) {
...
+				}
+
+				if execTask.Status == "failed" {
```
Comment (ablack12) [inline trigger/task_jira.go:110]: you can use evergreen.TaskFailed instead of hardcoding it

File: trigger/task_jira.go [hunk 4]
```diff
@@ -71,42 +69,63 @@ func getTaskURL(data *jiraTemplateData) (string, error) {
...
+			return result, nil
+		}
+	} else {
+		// Task is not display only
+		id := data.Task.Id
+		execution := data.Task.Execution
+		if len(data.Task.OldTaskId) != 0 {
+			id = data.Task.OldTaskId
+		}
+		return []taskInfo{{Tests: "Display", URL: taskLogLink(data.UIRoot, id, execution)}}, nil
```
Comment (ablack12) [inline trigger/task_jira.go:127]: Why are we setting Tests equal to "Display"?  I would think this should be the task's display name (although it isn't super clear to me what "Tests" is for)

**Review by ablack12 — CHANGES_REQUESTED** [review]

**Review by malikchaya2 — APPROVED** [review]

File: trigger/task_jira.go (hunk 1 above)
Comment (bynn) [reply trigger/task_jira.go]: done

**Review by ablack12 — APPROVED** [review]

**Pull request merged**